packages/salesforcedx-vscode-core/src/commands/forceCreateManifest.ts

```diff
@@ -62,7 +61,15 @@ export async function forceCreateManifest(
   const sourcePaths = uris.map(uri => uri.fsPath);
   const inputOptions = {
```

Comment:
The way this command is written if very different from the others. Will need a lot of restructuring to make it work like the other commands.